### PR TITLE
Move "ui::Helper::DoLayout"  to "CreateNode"。

### PR DESCRIPTION
**Pull request by chengstory**

### CODE REVIEW

**Comment by pandamicro** [comment]
What's the purpose of this modification? Which problem does it solve ?

File: cocos/editor-support/cocostudio/ActionTimeline/CSLoader.cpp [hunk 1]
```diff
@@ -265,17 +265,24 @@ Node* CSLoader::createNode(const std::string& filename)
...
-        return load->createNodeFromJson(filename);
+        node = load->createNodeFromJson(filename);
     }
-    
-    return nullptr;
+
+    if (node != nullptr)
+    {
+        Size frameSize = Director::getInstance()->getVisibleSize();
+        node->setContentSize(frameSize);
```
Comment (pandamicro) [inline cocos/editor-support/cocostudio/ActionTimeline/CSLoader.cpp]: How do you know user want to set the content size as visible size ?

**Comment by pandamicro** [comment]
@chengstory Please also assign somebody to modify the web engine to support the same API

**Comment by chengstory** [comment]
This is H5 pr: https://github.com/cocos2d/cocos2d-html5/pull/3113

**Pull request merged**